### ISSUE TITLE
[SUGGESTION] Ingame timer

### ISSUE THREAD

**Issue by oStarist** [nzp-team/nzportable#343]
**Describe your suggestion**
An overall timer in the top right corner of the game using the same font and text color as the points.

**Expected behavior**
It should only add time when the game is not paused.
It should reset when you either die or quit out of the current map.

The format should be: `HH:MM:SS`. Add milliseconds if you want but I think it would just be too much.

**Screenshots/Footage**
Basically like in this guy's "FIVE" game: https://youtu.be/MvTyAw3vkwk?si=Se9m7tiUl5UA5-6D

**Additional context**
If you want gameplay of the timer, the guy's gameplay has plenty of it.

**Comment by MotoLegacy** [comment]
I like it. I'll add this tomorrow.

**Comment by MotoLegacy** [comment]
Added to FTE in https://github.com/nzp-team/quakec/commit/6d37aaae5f251f096d9cc7686f1cbfb38b165cac, toggle-able with the new `scr_serverstopwatch` cvar. Will add to other platforms eventually.
